modules/contentbox-admin/layouts/inc/HTMLBodyEnd.cfm

```diff
@@ -15,7 +15,7 @@
         <script type="application/javascript" src="#prc.cbroot#/includes/js/#js#.js"></script>
     </cfloop>
     <cfloop list="#event.getValue( "jsFullAppendList", "", true )#" index="js">
-        <script type="application/javascript" src="#prc.cbroot#/includes/js/#js#.js"></script>
+        <script type="application/javascript" src="#js#.js"></script>
```

Comment:
I thought you where going to remove the extension? Also, maybe change the `script` to html 5 compliance